.travis.yml

```diff
@@ -9,7 +9,7 @@ script:
   - ./gradlew check --info
   - ./testutils/check-baselines.sh
   - go test testutils/check-calling-form-in-baseline/*.go
-  - go run $(find testutils/check-calling-form-in-baseline -type f | grep -v _test) -yaml src/test/java/com/google/api/codegen/testsrc/libraryproto/config_not_annotated/library_gapic.yaml src/test
+  - go run $(find testutils/check-calling-form-in-baseline -type f | grep -v _test) -yaml src/test/java/com/google/api/codegen/testsrc/libraryproto/config_not_annotated/testdata/library_gapic.yaml src/test
```

Comment:
I'm not sure this is consistent with what we do elsewhere in the codebase. We have a testsrc directory (which this is already under) for files that are inputs to tests, and a testdata directory for baseline files. So I'm not sure that introducing a new testdata folder here makes sense. Is there an example in the existing codebase that this is matching?

Reply:
I had seen `packagegen/testdata`, `metacode/testdata`, `go/testdata`, `advising/testdata`, and `config/testdata` and assumed it was a thing.

This is a nice setup because there are multiple library.protos in this repo, and say for packagegen tests, we can just test the specific library.proto located at `packagegen/testdata/library.proto`, and have the test class PackageTest.java be in `packagegen`

Reply:
Ok, moved GapicCodeGeneratorAnnotationsTest and its test resources into a new package as described above (except calling the package `protonannotations` instead of `config_not_annotated`.

I can move the other test resources for the gapic tests in a different PR.